Extensions/DebuggerTools/debuggertools.ts

```diff
@@ -13,6 +13,20 @@ namespace gdjs {
         runtimeScene.getGame().pause(true);
       };
 
+      /**
+       * Logs a message to the console.
+       * @param message - The message to log.
+       * @param type - The type of log (info, warning or error).
+       * @param group - The group of messages it belongs to.
+       */
+      export const log = function (
+        message: string,
+        type: 'info' | 'warning' | 'error',
+        group: string
+      ) {
+        gdjs.log(group, message, type, false);
```

Comment:
What do you think of dropping the last "internal" parameter?
Nothing is really internal or not in a sense, and this would make gdjs.log more lightweight to use (even with a default parameter).

Reply:
The internal is used to differentiate logs made by the engine from the ones made by the user through the events sheet, to filter them out later. I thought that it might be useful when debugging something to be able to focus on your own logs without having to manually filter out the groups of the internal logs. Bouh also commented before
> I think we should also filter the messages coming from the engine and those added by the user.

So the idea cannot be that bad if we both had it?

Reply:
Ok fair enough then!